src/main/java/LittleMissHelpful.java

```diff
@@ -0,0 +1,241 @@
+                            for (int i = 0; i < counter; i++) {
+                                int listNumber = i + 1;
+                                Task listItem = list.get(i);
+                                System.out.println(listNumber + ". " + listItem.toString());
+                            }
+                        }
+                        System.out.println(lineBreak);
+                        continue;
+                    }
+
+                    // Invalid command --> throw exception
+                    System.out.println(lineBreak);
+                    System.out.println("Alamak! Format salah already. Please provide a valid command.");
+                    System.out.println(lineBreak);
+                    continue;
+                }
+
+                String item = inputList[1];
+
+                // Check if user command is "mark" / "unmark" -> mark / unmark task
+                if (command.equalsIgnoreCase("mark")) {
+                    try {
+                        int listIndex = Integer.parseInt(item) - 1;
+                        if (listIndex < 0 || listIndex >= counter) {
+                            throw new TaskNotFoundException(lineBreak + "\nHai-ya, task number out of range lah.\n" + lineBreak);
+                        }
+                        Task curTask = list.get(listIndex);
+                        list.set(listIndex, curTask.markTask());
+                        System.out.println(lineBreak);
+                        System.out.println("Wah upz! You have marked this task as done: " + curTask.toString());
+                        System.out.println(lineBreak);
+                    } catch (NumberFormatException e) {
+                        System.out.println(lineBreak + "\nHai-yo, task number must be an integer lah.\n" + lineBreak);
+                    } catch (TaskNotFoundException e) {
+                        System.out.println(e.getMessage());
+                    }
+                    continue;
+
+                } else if (command.equalsIgnoreCase("unmark")) {
+                    try {
+                        int listIndex = Integer.parseInt(item) - 1;
+                        if (listIndex < 0 || listIndex >= counter) {
+                            throw new TaskNotFoundException(lineBreak + "Hai-ya, task number out of range lah.\n" + lineBreak);
+                        }
+                        Task curTask = list.get(listIndex);
+                        list.set(listIndex, curTask.unmarkTask());
+                        System.out.println(lineBreak);
+                        System.out.println("Ok, I see you laze. You have marked this task as not done yet: " + curTask.toString());
+                        System.out.println(lineBreak);
+                    } catch (NumberFormatException e) {
+                        System.out.println(lineBreak + "\nHai-yo, task number must be an integer lah.\n" + lineBreak);
```

Comment:
This check looks the same as the check for the "mark" command, perhaps can refactor to reduce repetition?